docs/product-user-guide/src/main/asciidoc/guided-decision-table-graphs-con.adoc

```diff
@@ -0,0 +1,14 @@
+[#guided_decision_table_graphs_con]
+= Guided Decision Table Graphs
+
+A Guided Decision Table Graph is a collection of related guided decision tables that are displayed within a single editor. You can use this editor to better visualize and work with various related decision tables in one location. Additionally, when a condition or an action in one table uses the same data type as a condition or an action in another table, the tables will be physically linked with a line in the table graph editor.
+
+For example, if one decision table determines a loan application rate and another table uses the application rate to determine some other action, then the two decision tables are linked in a guided decision table graph.
```

Comment:
then the two decision tables are linked in a guided decision table graph >>
"then both the decision tables are linked in a guided decision table graph."

Reply:
I'd actually argue that it's better to say "the two graphs are linked" vs. "both graphs are linked" in this case. To me, "both" sounds comprehensive, like "all" but for when there are just two options, often used emphatically. But table graphs can have many decision tables. Two could be linked, three others could be linked, and two others linked. So three separate instances of linked tables within a graph, for instance. I'm trying to just say that these two would be linked, not to say that others necessarily will be, if there are any. Not sure if I'm explaining myself well, but either way, I think "two" is better than "both" here.